### DISCUSSION TITLE
Audio and Video are desynchronized with --export clip-sequence

### DISCUSSION THREAD

**Discussion by eromawyn** [WyattBlue/auto-editor#533] [4. Bugs]
### Bug description

I tried to exports "clips" from rush files ... but the audio and video are desynchronized. The audio was correctly cut, but the video was desynchronized. Does auto-editor take care of the audio shift my original file has ??

Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'facecam.mov':
  Metadata:
    major_brand     : qt
    minor_version   : 512
    compatible_brands: qt
    encoder         : Lavf60.10.100
  Duration: 00:18:25.50, **start: 0.529000**, bitrate: 172699 kb/s
  Stream #0:0[0x1]: Video: mjpeg (Baseline) (jpeg / 0x6765706A), yuvj420p(pc, bt470bg/unknown/unknown), 3840x2160 [SAR 1:1 DAR 16:9], 172017 kb/s, 59.98 fps, 60 tbr, 10000k tbn (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : FFMP
  Stream #0:1[0x2]: Audio: pcm_s16le (sowt / 0x74776F73), 48000 Hz, mono, s16, 768 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]

### What's your platform?

```shell
OS: Windows 10 amd64
Python: 3.12.5
PyAV: 12.3.0 (GPL version 3 or later)
Auto-Editor: 25.1.0
```

### What command did you use

```shell
auto-editor.exe facecam.mov --export  clip-sequence
```

### What properties does your input video have?

facecam.mov:
 - recommendedTimebase: 2999/50
 - video:
   - track 0:
     - codec: mjpeg
     - fps: 1776792190/29621829
     - resolution: 3840x2160
     - aspect ratio: 16:9
     - pixel aspect ratio: 1
     - duration: 1104.9217516
     - pix fmt: yuvj420p
     - color range: 2 (pc)
     - color space: 5
     - timebase: 1/10000000
     - bitrate: 172017236
 - audio:
   - track 0:
     - codec: pcm_s16le
     - layout: mono
     - samplerate: 48000
     - duration: 1105.5
     - bitrate: 768000
 - container:
   - duration: 1105.5
   - bitrate: 172699260

Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'facecam.mov':
  Metadata:
    major_brand     : qt
    minor_version   : 512
    compatible_brands: qt
    encoder         : Lavf60.10.100
  Duration: 00:18:25.50, start: 0.529000, bitrate: 172699 kb/s
  Stream #0:0[0x1]: Video: mjpeg (Baseline) (jpeg / 0x6765706A), yuvj420p(pc, bt470bg/unknown/unknown), 3840x2160 [SAR 1:1 DAR 16:9], 172017 kb/s, 59.98 fps, 60
 tbr, 10000k tbn (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : FFMP
  Stream #0:1[0x2]: Audio: pcm_s16le (sowt / 0x74776F73), 48000 Hz, mono, s16, 768 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]

**Comment by WyattBlue** [comment]
Does this only happen with  `--export clip-sequence`?

**Comment by Cfomodz** [comment]
Remux your input file with FFmpeg to reset the start timestamp before processing with Auto-Editor:

```bash
ffmpeg -i input.mov -c copy -avoid_negative_ts make_zero output.mov
auto-editor output.mov --export clip-sequence
```

This command copies all streams without re-encoding while normalizing the timestamp to zero, preventing desynchronization during clip export.
